addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,395 @@
+        trigger: 'button[id="o_payment_form_pay"]:visible:not(:disabled)',
+    },
+    {
+        content: "Open Dropdown for See quotation",
+        extra_trigger: '.oe_cart .oe_website_sale_tx_status',
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "My account",
+        extra_trigger: '#top_menu li.dropdown .js_usermenu.show',
+        trigger: '#top_menu .dropdown-menu a[href="/my/home"]:visible',
+    },
+    {
+        content: "See Quotations",
+        trigger: '.o_portal_docs a:contains("Quotations")',
+    },
+
+    // enable extra step on website checkout and check extra step on checkout process
+    {
+        content: "Open Dropdown for logout",
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "Logout",
+        trigger: '#o_logout:contains("Logout")',
+    },
+    {
+        content: "Sign in as admin",
+        trigger: '#top_menu li a b:contains("Sign in")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function () {
+            $('.oe_login_form input[name="login"]').val("admin");
+            $('.oe_login_form input[name="password"]').val("admin");
+            $('.oe_login_form input[name="redirect"]').val("/shop/cart");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Open Customize menu",
+        trigger: '.o_menu_sections a:contains("Customize")',
+    },
+    {
+        content: "Enable Extra step",
+        trigger: 'a.dropdown-item label:contains("Extra Step Option")',
+    },
+    {
+        content: "Open Dropdown for logout",
+        extra_trigger: '.progress-wizard-step:contains("Extra Info")',
+        trigger: '#top_menu li.dropdown:visible a:contains("Mitchell Admin")',
+    },
+    {
+        content: "Logout",
+        trigger: '#o_logout:contains("Logout")',
+    },
+    {
+        content: "Sign in as abc",
+        trigger: '#top_menu li a b:contains("Sign in")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function () {
+            $('.oe_login_form input[name="login"]').val("abc@odoo.com");
+            $('.oe_login_form input[name="password"]').val("1admin@admin");
+            $('.oe_login_form input[name="redirect"]').val("/shop?search=Storage Box");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Open product page",
+        trigger: '.oe_product_cart a:contains("Storage Box")',
+    },
+    {
+        content: "Click on add to cart",
+        trigger: '#add_to_cart',
+    },
+    {
+        content: "Proceed to checkout",
+        trigger: 'a[href*="/shop/checkout"]',
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Select `Wire Transfer` payment method",
```

Comment:
Can you ensure the address here is the one added during the second order?

Reply:
👍